pyartcd/pyartcd/pipelines/buils_sync.py

```diff
@@ -0,0 +1,92 @@
+import time
+import click
+import logging
+from pyartcd import constants, exectools
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@cli.group("imagestreams", short_help="Manage ART equivalent images in upstream CI.")
+async def build_sync()
+    pass
+
+
+@build_sync.command('backup', short_help='Reads streams.yml and mirrors out ART equivalent images to api.ci.')
+@pass_runtime
+@click_coroutine
+async def backup_imagestream():
+    allNameSpaces = ["ocp", "ocp-priv", "ocp-s390x", "ocp-s390x-priv", "ocp-ppc64le", "ocp-ppc64le-priv", "ocp-arm64", "ocp-arm64-priv"]
+    tar = tarfile.open("app.ci-backup.tgz","w:gz")
+    for ns in allNameSpaces:
+        cmd = f"oc --kubeconfig {constants.CIKUBECONFIG} get is -n {ns} -o yaml"
+        _LOGGER.info("Running command: %s", cmd)
+        _, stdout, _  = await exectools.cmd_gather_async(cmd)
+        async with aiofiles.open(f"${ns}.backup.yaml", "w") as f:
+            await f.write(stdout.read())
+        tar.add(f"${ns}.backup.yaml")
+
+        cmd = f"oc --kubeconfig {constants.CIKUBECONFIG} get secret/release-upgrade-graph -n ${ns} -o yaml"
+        _LOGGER.info("Running command: %s", cmd)
+        _, stdout, _  = await exectools.cmd_gather_async(cmd)
+        async with aiofiles.open(f"${ns}.release-upgrade-graph.backup.yaml", "w") as f:
+            await f.write(stdout.read())
+        tar.add(f"${ns}.release-upgrade-graph.backup.yaml")
+    tar.close()
+
+
+@build_sync.command('trigger', short_help='Reads streams.yml and mirrors out ART equivalent images to api.ci.')
+@click.option('--version', metavar='VERSION', required=True, help='OCP version')
+@pass_runtime
+@click_coroutine
+async def trigger_release(version, dry_run):
+    cmd = f"oc --kubeconfig {constants.CIKUBECONFIG} -n ocp tag registry.access.redhat.com/ubi8 {version}-art-latest:trigger-release-controller"
+    _LOGGER.info("Running command: %s", cmd)
+    await exectools.cmd_assert_async(cmd)
+    time.sleep(10)
```

Comment:
`time.sleep` can't be used in an `async` function. Can be replaced with `await asyncio.sleep()`.